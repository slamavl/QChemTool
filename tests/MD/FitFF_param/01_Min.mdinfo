

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2022       1.4598E+03     9.9439E-05     4.0986E-04     F178      307

 BOND    =      192.2475  ANGLE   =      188.0655  DIHED      =      673.9530
 VDWAALS =      300.1170  EEL     =       12.6589  HBOND      =        0.0000
 1-4 VDW =      228.9057  1-4 EEL =     -136.1429  RESTRAINT  =        0.0000
