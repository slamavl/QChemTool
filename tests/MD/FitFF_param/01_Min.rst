FGR 
  330
 -10.5989503  -5.2374172   0.2651347 -10.6919938  -5.2691924   1.6103248
 -11.7803537  -5.2626660  -0.3864176  -9.8352551  -6.5601704  -0.2654966
 -10.4478494  -7.5706774   0.3860216  -9.9092571  -6.6248383  -1.6106899
  -9.7617082  -3.9346844  -0.2317208  -9.8404784  -3.9312699  -1.6966837
 -10.5737861  -2.6243007   0.2707136 -10.6574118  -2.6146373   1.6150671
 -11.7590409  -2.6402089  -0.3716095  -9.7614173  -1.3101881  -0.2225409
  -9.8529232  -1.2900447  -1.6864238 -10.5724971  -0.0016294   0.2919303
 -10.6448687   0.0146187   1.6368749 -11.7634681  -0.0176451  -0.3397001
  -9.7624349   1.3101680  -0.2089144  -9.8604717   1.3265120  -1.6722800
 -10.5678824   2.6251339   0.3055927 -10.6305681   2.6536591   1.6508856
 -11.7643840   2.6066477  -0.3157241  -9.7640204   3.9345335  -0.2122760
  -9.8913475   3.9248115  -1.6734005 -10.5679628   5.2471734   0.3335974
 -10.5499246   5.3325192   1.6784474 -11.8069377   5.2383922  -0.1979549
  -9.8281185   6.5286158  -0.3335184  -9.8930216   6.4703277  -1.6783681
 -10.4399887   7.6059925   0.1980434  -8.2893638   6.4886870   0.2123430
  -7.5573149   7.8395477  -0.3055272  -7.6133709   7.8795770  -1.6508196
  -8.1395434   8.8849942   0.3157971  -8.3445962   6.6038122   1.6734683
  -7.5252221   5.2065475  -0.2197191  -7.5490510   5.2175121  -1.6836472
  -8.2715814   3.9138235   0.2197761  -8.2929817   3.9289721   1.6837045
  -7.5270839   2.6080170  -0.2065898  -7.5515361   2.6099432  -1.6701884
  -8.2730476   1.3051412   0.2267316  -8.3037116   1.3079639   1.6905161
  -7.5276192  -0.0007100  -0.2042560  -7.5484395   0.0023353  -1.6678395
  -8.2745802  -1.3076521   0.2214808  -8.3115964  -1.3100613   1.6851086
  -7.5283655  -2.6110349  -0.2094348  -7.5471156  -2.6110300  -1.6731148
  -8.2768765  -3.9172771   0.2145441  -8.3207147  -3.9269319   1.6780275
  -7.5309346  -5.2092845  -0.2148196  -7.5611752  -5.2423989  -1.6783044
  -8.2884480  -6.4864839   0.2314044  -8.3249182  -6.5564300   1.6963652
  -7.5596577  -7.8449540  -0.2710294  -7.5930647  -7.9221875  -1.6153851
  -8.1660881  -8.8634654   0.3712615  -6.0154321  -7.7984974   0.2222696
  -6.0437820  -7.8878356   1.6861505  -5.2877231  -9.1551912  -0.2921994
  -5.3098013  -9.2259726  -1.6371456  -5.8971040 -10.1785982   0.3394005
  -3.7466556  -9.1095741   0.2086886  -3.7815604  -9.2026670   1.6720521
  -3.0105822 -10.4645944  -0.3058151  -3.0171864 -10.5331278  -1.6511090
  -3.6248670 -11.4915548   0.3154728  -1.4746913 -10.4231475   0.2120951
  -1.5468136 -10.5285721   1.6732163  -0.7398794 -11.7757015  -0.3337749
  -0.6569147 -11.8027381  -1.6786230  -1.3669934 -12.8442957   0.1977494
   0.7397875 -11.7757177   0.3333786   0.6568225 -11.8027961   1.6782259
   1.3668932 -12.8443001  -0.1981793   1.4746097 -10.4231521  -0.2124490
   3.0105008 -10.4646277   0.3054580   3.0171059 -10.5332048   1.6507497
   3.6247769 -11.4915728  -0.3158639   1.5467298 -10.5285302  -1.6735737
   0.7463146  -9.1203158   0.2196109   0.7438584  -9.1464528   1.6835387
  -0.7463858  -9.1203029  -0.2199223  -0.7439297  -9.1463920  -1.6838509
  -1.5049996  -7.8226454   0.2064405  -1.5155962  -7.8448038   1.6700385
  -3.0062934  -7.8172135  -0.2269206  -3.0191424  -7.8451610  -1.6907058
  -3.7644809  -6.5187280   0.2040642  -3.7722935  -6.5383016   1.6676472
  -5.2697949  -6.5121263  -0.2217134  -5.2903501  -6.5429573  -1.6853421
  -6.0254518  -5.2141944   0.2092005  -6.0348627  -5.2304563   1.6728800
  -5.2724006  -3.9122470  -0.2146396  -5.2803745  -3.9171629  -1.6783294
  -6.0243317  -2.6098656   0.2144451  -6.0326146  -2.6143356   1.6781346
  -5.2720715  -1.3051161  -0.2111737  -5.2777579  -1.3047732  -1.6748095
  -6.0229814  -0.0009125   0.2171112  -6.0285907  -0.0015733   1.6807947
  -5.2710886   1.3034135  -0.2111768  -5.2772379   1.3035942  -1.6748174
  -6.0218826   2.6081724   0.2147306  -6.0257332   2.6121108   1.6784215
  -5.2696556   3.9110590  -0.2146922  -5.2750001   3.9124293  -1.6783830
  -6.0221089   5.2146839   0.2066368  -6.0359934   5.2348930   1.6702355
  -5.2667599   6.5121405  -0.2266861  -5.2845455   6.5372889  -1.6904704
  -6.0157938   7.7994793   0.2089680  -6.0789553   7.8762071   1.6723342
  -5.2847680   9.1569075  -0.2918798  -5.3350337   9.2114617  -1.6368240
  -5.8663710  10.1963301   0.3397562  -3.7459798   9.1087585   0.2225803
  -3.0141018  10.4693413  -0.2706784  -3.0642909  10.5369326  -1.6150316
  -3.5929405  11.5037605   0.3716485  -3.8091671   9.1779323   1.6864638
  -3.0047704   7.8198465  -0.2214475  -3.0212001   7.8531104  -1.6850751
  -3.7631426   6.5194935   0.2042918  -3.7761807   6.5359991   1.6678754
  -3.0106610   5.2165352  -0.2170825  -3.0129013   5.2217269  -1.6807660
  -3.7643018   3.9132188   0.2112065  -3.7675246   3.9184501   1.6748472
  -3.0118780   2.6084971  -0.2136314  -3.0128927   2.6086564  -1.6772584
  -3.7649525   1.3041431   0.2136534  -3.7655903   1.3049378   1.6772804
  -3.0126440  -0.0004033  -0.2128025  -3.0141674  -0.0003100  -1.6764219
  -3.7658422  -1.3049643   0.2136820  -3.7669950  -1.3063898   1.6773091
  -3.0130682  -2.6088062  -0.2137977  -3.0148413  -2.6090700  -1.6774248
  -3.7663356  -3.9131635   0.2110191  -3.7689206  -3.9182807   1.6746548
  -3.0123160  -5.2155721  -0.2172646  -3.0156547  -5.2200786  -1.6809483
  -1.5068014  -5.2165946   0.2110627  -1.5097577  -5.2220308   1.6747032
  -0.7522432  -6.5191808  -0.2148431  -0.7507204  -6.5244649  -1.6785340
   0.7521923  -6.5191939   0.2146180   0.7506696  -6.5245273   1.6783088
   1.5049382  -7.8226502  -0.2067095   1.5155334  -7.8447600  -1.6703083
   3.0062326  -7.8172446   0.2266502   3.0190828  -7.8452413   1.6904344
   3.7465840  -9.1095965  -0.2090031   3.7814853  -9.2026413  -1.6723697
   5.2876522  -9.1552423   0.2918798   5.3097326  -9.2260691   1.6368235
   5.8970238 -10.1786328  -0.3397558   6.0153705  -7.7985368  -0.2225455
   7.5595972  -7.8450225   0.2707472   7.5930072  -7.9223034   1.6151000
   8.1660178  -8.8635162  -0.3715811   6.0437159  -7.8878254  -1.6864295
   5.2697444  -6.5121751   0.2214833   5.2903018  -6.5430570   1.6851109
   3.7644297  -6.5187504  -0.2042919   3.7722397  -6.5382743  -1.6678756
   3.0122754  -5.2156031   0.2170824   3.0156151  -5.2201600   1.6807660
   1.5067605  -5.2165991  -0.2112439   1.5097158  -5.2219854  -1.6748846
   0.7530497  -3.9126204   0.2135935   0.7526431  -3.9135996   1.6772205
  -0.7530803  -3.9126071  -0.2137297  -0.7526736  -3.9135359  -1.6773567
  -1.5066972  -2.6088159   0.2127257  -1.5074158  -2.6102032   1.6763451
  -0.7532095  -1.3044424  -0.2132655  -0.7533659  -1.3045070  -1.6768816
  -1.5062904  -0.0000687   0.2132271  -1.5064618  -0.0001934   1.6768432
  -0.7530765   1.3045266  -0.2132199  -0.7530612   1.3047414  -1.6768360
  -1.5059513   2.6092391   0.2128169  -1.5067864   2.6105080   1.6764362
  -0.7527593   3.9138036  -0.2136682  -0.7521082   3.9155181  -1.6772953
  -1.5057302   5.2183173   0.2111942  -1.5088624   5.2230674   1.6748300
  -0.7519060   6.5221633  -0.2144261  -0.7521832   6.5315739  -1.6781156
  -1.5028979   7.8252848   0.2094594  -1.5122688   7.8415188   1.6731395
  -0.7459065   9.1266298  -0.2145224  -0.7594709   9.1694232  -1.6780057
  -1.4732347  10.4212410   0.2317467  -1.5155679  10.4877511   1.6967098
  -0.7636479  11.7976745  -0.2651134  -0.7826573  11.8941393  -1.6103034
  -1.3324724  12.8334293   0.3864408   0.7637400  11.7976603   0.2655108
   0.7827500  11.8940826   1.6107039   1.3325726  12.8334310  -0.3860109
   1.4733164  10.4212369  -0.2313923   3.0141833  10.4693090   0.2710364
   3.0643712  10.5368560   1.6153918   3.5930307  11.5037445  -0.3712562
   1.5156516  10.4877938  -1.6963533   0.7459777   9.1266169   0.2148343
   0.7595424   9.1693622   1.6783191   1.5029592   7.8252800  -0.2091900
   1.5123316   7.8415627  -1.6728695   0.7519569   6.5221502   0.2146514
   0.7522341   6.5315113   1.6783413   1.5057710   5.2183127  -0.2110128
   1.5089043   5.2231129  -1.6746485   0.7527899   3.9137903   0.2138044
   0.7521387   3.9154542   1.6774315   1.5059717   2.6092348  -0.2127257
   1.5068073   2.6105545  -1.6763450   0.7530867   1.3045133   0.2132655
   0.7530714   1.3046769   1.6768816   1.5062904  -0.0000730  -0.2132271
   1.5064618  -0.0001466  -1.6768432   0.7531993  -1.3044557   0.2132199
   0.7533557  -1.3045714   1.6768360   1.5066767  -2.6088203  -0.2128169
   1.5073948  -2.6101567  -1.6764363   3.0130479  -2.6088372   0.2137060
   3.0148215  -2.6091524   1.6773331   3.7663049  -3.9131855  -0.2111569
   3.7688883  -3.9182519  -1.6747928   5.2723703  -3.9122958   0.2145001
   5.2803457  -3.9172636   1.6781898   6.0254107  -5.2142341  -0.2093869
   6.0348186  -5.2304450  -1.6730670   7.5308943  -5.2093511   0.2146303
   7.5611375  -5.2425183   1.6781138   8.2883966  -6.4865404  -0.2316412
   8.3248619  -6.5564350  -1.6966045   9.8352048  -6.5602569   0.2652521
   9.9092105  -6.6249746   1.6104427  10.4477891  -7.5707447  -0.3863051
  10.5989085  -5.2374902  -0.2653331  10.6919477  -5.2692166  -1.6105246
  11.7803138  -5.2627721   0.3862146   9.7616779  -3.9347693   0.2315731
  10.5737652  -2.6243731  -0.2708136  10.6573890  -2.6146604  -1.6151669
  11.7590208  -2.6403145   0.3715071   9.8404510  -3.9314099   1.6965359
   8.2768455  -3.9173340  -0.2146883   8.3206808  -3.9269364  -1.6781722
   7.5283454  -2.6111014   0.2093392   7.5470969  -2.6111501   1.6730192
   6.0243111  -2.6099051  -0.2145391   6.0325925  -2.6143231  -1.6782288
   5.2720614  -1.3051649   0.2111269   5.2777483  -1.3048745   1.6747627
   3.7658320  -1.3049862  -0.2137283   3.7669842  -1.3063602  -1.6773554
   3.0126440  -0.0004344   0.2128024   3.0141675  -0.0003927   1.6764218
   3.7649627   1.3041213  -0.2136074   3.7656010   1.3049675  -1.6772344
   3.0118983   2.6084660   0.2137230   3.0129124   2.6085739   1.6773501
   3.7643325   3.9131967  -0.2110687   3.7675568   3.9184789  -1.6747092
   3.0107015   5.2165042   0.2172648   3.0129408   5.2216452   1.6809485
   3.7631938   6.5194711  -0.2040639   3.7762343   6.5360264  -1.6676469
   3.0048312   7.8198156   0.2217184   3.0212597   7.8530300   1.6853471
   3.7460514   9.1087366  -0.2222653   3.8092422   9.1779584  -1.6861464
   5.2848388   9.1568563   0.2922001   5.3351022   9.2113646   1.6371463
   5.8664513  10.1962957  -0.3393995   6.0158553   7.7994394  -0.2086917
   7.5573753   7.8394779   0.3058094   7.6134280   7.8794592   1.6511034
   8.1396136   8.8849417  -0.3154763   6.0790211   7.8762169  -1.6720551
   5.2668103   6.5120915   0.2269164   5.2845938   6.5371888   1.6907016
   6.0221500   5.2146441  -0.2064505   6.0360373   5.2349043  -1.6700484
   5.2696859   3.9110102   0.2148315   5.2750289   3.9123285   1.6785224
   6.0219032   2.6081330  -0.2146369   6.0257552   2.6121235  -1.6783276
   5.2710988   1.3033648   0.2112233   5.2772476   1.3034929   1.6748640
   6.0229815  -0.0009517  -0.2171113   6.0285907  -0.0015602  -1.6807949
   7.5276193  -0.0007763   0.2042558   7.5484396   0.0022150   1.6678394
   8.2745699  -1.3077086  -0.2215293   8.3115851  -1.3100648  -1.6851571
   9.7614072  -1.3102728   0.2224914   9.8529142  -1.2901851   1.6863750
  10.5724971  -0.0017010  -0.2919304  10.6448688   0.0145967  -1.6368744
  11.7634680  -0.0177497   0.3396993   9.7624451   1.3100839   0.2089636
   9.8604811   1.3263721   1.6723299   8.2730580   1.3050849  -0.2266836
   8.3037230   1.3079608  -1.6904679   7.5271041   2.6079507   0.2066850
   7.5515550   2.6098232   1.6702836   8.2716125   3.9137669  -0.2196324
   8.2930155   3.9289682  -1.6835601   7.5252623   5.2064808   0.2199081
   7.5490885   5.2173925   1.6838367   8.2894151   6.4886298  -0.2121063
   8.3446523   6.6038064  -1.6732273   9.8281685   6.5285268   0.3337612
   9.8930673   6.4701889   1.6786089  10.4400486   7.6059183  -0.1977593
  10.5680048   5.2471031  -0.3333995  10.5499711   5.3324983  -1.6782464
  11.8069781   5.2382928   0.1981561   9.7640508   3.9344493   0.2124233
   9.8913751   3.9246722   1.6735477  10.5679035   2.6250627  -0.3054930
  11.7644039   2.6065441   0.3158250  10.6305914   2.6536374  -1.6507847
