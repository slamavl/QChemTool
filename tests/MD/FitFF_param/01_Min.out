
          -------------------------------------------------------
          Amber 14 SANDER                              2014
          -------------------------------------------------------

| Run on 03/13/2018 at 16:46:38

|   Executable path: sander
| Working directory: /mnt/sda2/Dropbox/PhD/Programy/Python/QChemTool/tests/MD/FitFF_param
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 01_Min.in                                                             
| MDOUT: 01_Min.out                                                            
|INPCRD: FGrph_reord.inpcrd                                                    
|  PARM: FGrph_reord.prmtop                                                    
|RESTRT: 01_Min.rst                                                            
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: 01_Min.mdcrd                                                          
|MDINFO: 01_Min.mdinfo                                                         
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

minimize                                                                       
 &cntrl                                                                        
  imin=1,                                                                      
  ntb=0,                                                                       
  ntx=1,                                                                       
  irest=0,                                                                     
  maxcyc=10000,                                                                
  ncyc=4000,                                                                   
  ntpr=100,                                                                    
  ntwx=10,                                                                     
  cut=999.0,                                                                   
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
|    NONPERIODIC   ntb=0 and igb=0: Setting up nonperiodic simulation
|  *** cutoff > system size, list only builds once
|Largest sphere to fit in unit cell has radius =    31.728
| New format PARM file being parsed.
| Version =    1.000 Date = 03/13/18 Time = 16:46:38
 NATOM  =     330 NTYPES =       2 NBONH =       0 MBONA  =     390
 NTHETH =       0 MTHETA =     900 NPHIH =       0 MPHIA  =    3126
 NHPARM =       0 NPARM  =       0 NNB   =    2998 NRES   =       1
 NBONA  =     390 NTHETA =     900 NPHIA =    3126 NUMBND =       5
 NUMANG =      10 NPTRA  =       8 NATYP =       4 NPHB   =       0
 IFBOX  =       0 NMXRS  =     330 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real               30567
|     Hollerith            993
|     Integer           174685
|     Max Pairs          54285
|     nblistReal          3960
|     nblist Int       1528485
|       Total             7138 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

FGR                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     100, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =      10, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =   10000, ncyc    =    4000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

FGR                                                                             
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =  -0.00000002
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------

     eedmeth=4: Setting switch to one everywhere

 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =      51288
| TOTAL SIZE OF NONBOND LIST =      51288


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       1.6880E+03     2.8133E+01     1.0564E+02     C7         19

 BOND    =      346.9086  ANGLE   =      261.3907  DIHED      =      686.2505
 VDWAALS =      290.9881  EEL     =       13.3706  HBOND      =        0.0000
 1-4 VDW =      226.0755  1-4 EEL =     -137.0145  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100       1.4599E+03     6.4941E-02     2.5819E-01     F229      201

 BOND    =      192.2677  ANGLE   =      187.7565  DIHED      =      674.5941
 VDWAALS =      300.0397  EEL     =       12.6823  HBOND      =        0.0000
 1-4 VDW =      228.7762  1-4 EEL =     -136.1834  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200       1.4598E+03     4.9268E-02     1.7236E-01     C157      283

 BOND    =      192.0509  ANGLE   =      188.1993  DIHED      =      674.1438
 VDWAALS =      299.9650  EEL     =       12.6587  HBOND      =        0.0000
 1-4 VDW =      228.9435  1-4 EEL =     -136.1414  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300       1.4598E+03     1.0599E-02     3.9456E-02     F318       21

 BOND    =      192.1883  ANGLE   =      188.1280  DIHED      =      674.0347
 VDWAALS =      299.9833  EEL     =       12.6573  HBOND      =        0.0000
 1-4 VDW =      228.9587  1-4 EEL =     -136.1409  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400       1.4598E+03     1.3330E-02     7.4064E-02     C104      222

 BOND    =      192.2200  ANGLE   =      188.1164  DIHED      =      673.9974
 VDWAALS =      300.0084  EEL     =       12.6573  HBOND      =        0.0000
 1-4 VDW =      228.9488  1-4 EEL =     -136.1411  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500       1.4598E+03     1.1212E-02     4.9218E-02     C104      222

 BOND    =      192.2196  ANGLE   =      188.1224  DIHED      =      673.9797
 VDWAALS =      300.0305  EEL     =       12.6574  HBOND      =        0.0000
 1-4 VDW =      228.9376  1-4 EEL =     -136.1410  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600       1.4598E+03     2.1285E-03     8.8141E-03     F297       62

 BOND    =      192.2300  ANGLE   =      188.1070  DIHED      =      673.9726
 VDWAALS =      300.0484  EEL     =       12.6577  HBOND      =        0.0000
 1-4 VDW =      228.9313  1-4 EEL =     -136.1414  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700       1.4598E+03     1.7016E-03     6.8248E-03     F269      117

 BOND    =      192.2343  ANGLE   =      188.0981  DIHED      =      673.9678
 VDWAALS =      300.0629  EEL     =       12.6579  HBOND      =        0.0000
 1-4 VDW =      228.9258  1-4 EEL =     -136.1417  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800       1.4598E+03     1.3650E-03     5.6763E-03     F321       16

 BOND    =      192.2369  ANGLE   =      188.0915  DIHED      =      673.9645
 VDWAALS =      300.0744  EEL     =       12.6581  HBOND      =        0.0000
 1-4 VDW =      228.9215  1-4 EEL =     -136.1420  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900       1.4598E+03     1.8374E-03     6.4087E-03     C3          9

 BOND    =      192.2398  ANGLE   =      188.0853  DIHED      =      673.9620
 VDWAALS =      300.0834  EEL     =       12.6583  HBOND      =        0.0000
 1-4 VDW =      228.9182  1-4 EEL =     -136.1422  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000       1.4598E+03     2.0827E-03     7.8104E-03     C13        37

 BOND    =      192.2405  ANGLE   =      188.0821  DIHED      =      673.9601
 VDWAALS =      300.0905  EEL     =       12.6584  HBOND      =        0.0000
 1-4 VDW =      228.9155  1-4 EEL =     -136.1423  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1100       1.4598E+03     1.8867E-03     6.5851E-03     C59       135

 BOND    =      192.2428  ANGLE   =      188.0776  DIHED      =      673.9585
 VDWAALS =      300.0962  EEL     =       12.6585  HBOND      =        0.0000
 1-4 VDW =      228.9134  1-4 EEL =     -136.1425  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1200       1.4598E+03     1.6096E-03     6.9456E-03     C13        37

 BOND    =      192.2432  ANGLE   =      188.0756  DIHED      =      673.9573
 VDWAALS =      300.1009  EEL     =       12.6586  HBOND      =        0.0000
 1-4 VDW =      228.9116  1-4 EEL =     -136.1426  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1300       1.4598E+03     1.2880E-03     5.2663E-03     C13        37

 BOND    =      192.2448  ANGLE   =      188.0726  DIHED      =      673.9563
 VDWAALS =      300.1047  EEL     =       12.6587  HBOND      =        0.0000
 1-4 VDW =      228.9103  1-4 EEL =     -136.1427  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1400       1.4598E+03     1.1932E-03     4.8715E-03     C104      222

 BOND    =      192.2450  ANGLE   =      188.0712  DIHED      =      673.9555
 VDWAALS =      300.1077  EEL     =       12.6587  HBOND      =        0.0000
 1-4 VDW =      228.9091  1-4 EEL =     -136.1427  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500       1.4598E+03     3.0791E-04     1.2645E-03     F249      159

 BOND    =      192.2458  ANGLE   =      188.0696  DIHED      =      673.9549
 VDWAALS =      300.1101  EEL     =       12.6587  HBOND      =        0.0000
 1-4 VDW =      228.9082  1-4 EEL =     -136.1428  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1600       1.4598E+03     8.2976E-04     3.5506E-03     C13        37

 BOND    =      192.2461  ANGLE   =      188.0686  DIHED      =      673.9543
 VDWAALS =      300.1121  EEL     =       12.6588  HBOND      =        0.0000
 1-4 VDW =      228.9075  1-4 EEL =     -136.1428  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1700       1.4598E+03     8.1297E-04     3.4711E-03     C59       135

 BOND    =      192.2468  ANGLE   =      188.0673  DIHED      =      673.9539
 VDWAALS =      300.1137  EEL     =       12.6588  HBOND      =        0.0000
 1-4 VDW =      228.9069  1-4 EEL =     -136.1428  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1800       1.4598E+03     1.8471E-04     7.8111E-04     C150      305

 BOND    =      192.2470  ANGLE   =      188.0666  DIHED      =      673.9536
 VDWAALS =      300.1150  EEL     =       12.6588  HBOND      =        0.0000
 1-4 VDW =      228.9064  1-4 EEL =     -136.1429  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1900       1.4598E+03     1.2899E-04     5.3193E-04     F178      307

 BOND    =      192.2472  ANGLE   =      188.0661  DIHED      =      673.9533
 VDWAALS =      300.1160  EEL     =       12.6588  HBOND      =        0.0000
 1-4 VDW =      228.9061  1-4 EEL =     -136.1429  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000       1.4598E+03     1.0954E-04     4.6602E-04     C150      305

 BOND    =      192.2474  ANGLE   =      188.0655  DIHED      =      673.9531
 VDWAALS =      300.1168  EEL     =       12.6589  HBOND      =        0.0000
 1-4 VDW =      228.9058  1-4 EEL =     -136.1429  RESTRAINT  =        0.0000


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2022       1.4598E+03     9.9439E-05     4.0986E-04     F178      307

 BOND    =      192.2475  ANGLE   =      188.0655  DIHED      =      673.9530
 VDWAALS =      300.1170  EEL     =       12.6589  HBOND      =        0.0000
 1-4 VDW =      228.9057  1-4 EEL =     -136.1429  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.01% of Total)
|                Build the list             0.00 (85.93% of List )
|                Other                      0.00 (14.07% of List )
|             List time                  0.00 ( 0.05% of Nonbo)
|                   Short_ene time             3.28 (99.79% of Direc)
|                   Other                      0.01 ( 0.21% of Direc)
|                Direct Ewald time          3.29 (96.29% of Ewald)
|                Virial junk                0.00 ( 0.01% of Ewald)
|                Start synchronizatio       0.00 ( 0.01% of Ewald)
|                Other                      0.13 ( 3.69% of Ewald)
|             Ewald time                 3.41 (100.0% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force              3.42 (73.48% of Force)
|          Bond/Angle/Dihedral        1.23 (26.48% of Force)
|          Other                      0.00 ( 0.04% of Force)
|       Force time                 4.65 (100.0% of Runmd)
|    Runmd Time                 4.65 (97.41% of Total)
|    Other                      0.12 ( 2.58% of Total)
| Total time                 4.77 (100.0% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:       1595
| Highest istack allocated:        319
|           Job began  at 16:46:38.247  on 03/13/2018
|           Setup done at 16:46:38.255  on 03/13/2018
|           Run   done at 16:46:43.020  on 03/13/2018
|     wallclock() was called   46541 times
